SparkleDotNET/SUAppcastItem.cs

```diff
@@ -39,6 +39,7 @@ public class SUAppcastItem : IComparable<SUAppcastItem> {
                 ItemDescription = (string)dict.ValueForKey("description");
                 FileURL = new Uri((string)enclosure.ValueForKey("url"));
                 DSASignature = (string)enclosure.ValueForKey("sparkle:dsaSignature");
+                RSASignature = (string)enclosure.ValueForKey("sparkle:rsaSignature");

```

Comment:
Yeah. I've branched off of your `ancast` branch in btbrowser-jenkins-helpers to add the functionality to extract the sha1 "thumbprint" from our `mini_installer.exe`.